runtime/src/disable.rs

```diff
@@ -19,11 +20,32 @@ impl DisableAttestationCmd {
             .erase_key(KEY_ID_RT_PRIV_KEY)
             .map_err(|_| CaliptraError::RUNTIME_DISABLE_ATTESTATION_FAILED)?;
 
+        Self::reinitialize_cdi(drivers)?;
         Self::generate_dice_key(drivers)?;
         drivers.attestation_disabled = true;
         Ok(MailboxResp::default())
     }
 
+    // Reinitialize cdi key vault slot to an hmac of a buffer of 0s.
+    fn reinitialize_cdi(drivers: &mut Drivers) -> CaliptraResult<()> {
```

Comment:
nit: could you rename this to something like `zero_rt_cdi` to make it clear that the CDI slot no longer contains a secret tied to the UDS?

Reply:
Done